### DISCUSSION TITLE
比起很會寫 Code，善用「無程式碼工具」才是未來職場的必備條件？

### DISCUSSION THREAD

**Discussion by 32844583** [1101-GS3073-C/Python-Community-GS3073-C#512] [General]
### 「無程式碼平台」大幅簡化 Coding 流程
新世代的學生比以往更精通新科技，並且能夠快速上手使用工具進行分析和研究。他們不害怕學習新的工具和技術，並且能透過自學和網路搜索解決問題。此外，他們能夠使用無程式碼平台帶來的生產力，即使是很基礎但精確的數據分析程式也會帶來幫助。新世代的人比過去任何時代的人都更善於將不同的個人化服務整合起來，創造更好的體驗，並更有效地運用數據。
https://buzzorange.com/techorange/2020/12/23/no-code-age-is-coming/

**Comment by NcuJerryKao** [comment]
No code討論 的文章 我也看到蠻多的了

此點贊同 正所謂 寫程式 主要是培養思維能力 而不在於打出某一特定的程式語言'

**Comment by Ben6uo** [comment]
感覺可以把我那程式能力升一點了

**Comment by todoroki0910** [comment]
確實，透過自學和網路搜索解決問題，才能再迅速變遷的時代生存下來

**Comment by BingRong1217** [comment]
時代在變遷，那這是否代表學習程式語言不再重要?

**Comment by awkward-willy** [comment]
比起學習複雜、深奧且龐雜的程式概念，這些 No-code/Low-code 的工具確實能讓新手在一開始時快速上手並做出所需。但若要創造出更多彈性與創造特製化工具，還是需要一套完整的思維能力。

**Comment by williamyen0623** [comment]
想當初最一開始接觸程式也是用拉積木的方式，但代碼相較無代碼的圖形化開發工具更能客製化需求，不可否認的是無代碼的確省去不少因學習程式語法的時間，能成為即生產力

**Comment by limitx0** [comment]
如果不是應徵相關工作寫程式當然不是必備技能
但是不論什麼工作能善用工具的人一定更吃香

**Comment by Chenszuhong** [comment]
這個主題真是不錯的想法 感覺都很有趣而且很帥 想要知道更多能延生應用的地方

**Comment by Raywei0214** [comment]
看標題看成比起寫code，善用無碼工具才點進來的，無碼的出現，必將帶領人類走向更高的巔峰。每個人都不得不面對這些問題。在面對這種問題時，務必詳細考慮無碼的各種可能。

**Comment by larry40305** [comment]
現在很多行業都需要程式能力，這個產品應該可以有效幫助那些不會寫程式的人

**Comment by Yilin20021016** [comment]
但是這樣的話 讀CS的朋友會被分掉?

**Comment by HsuChen88** [comment]
曾經人們為了自動化各種工作的繁瑣行為，趨之若鶩的鑽研程式，如今又讓程式重回幕後，大家只需懂得操作GUI即可。

**Comment by xCk27x** [comment]
感覺這種no code的服務跟一般cs的工作深度還是有差
因為針對**客製化**的數據分析資料處理
常常無法靠這些no code的服務完成
仍需要專業技術人員去逐步進行開發

> 但如果只是一些簡單的工作
這種no code的服務就真的蠻好用的
主要還是看使用者串接不同平台的能力

**Comment by lx3211** [comment]
現在都可以叫AI寫程式碼，真正需要的是會善用工具的人
